### PR TITLE
Drop Katello upgrade steps for Mongo and Qpid certs

### PR DESCRIPTION
**Pull request by ehelms**
There have been 10 releases since the qpidd upgrade step was introduced and 8 since we upgraded Mongo. Therefore, I feel fairly confident we can drop these from our current upgrade steps.

@jlsherrill @chris1984

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]
Yeah most people should be off Mongo 2.x now, ACK for the Mongo side of things

**Comment by chris1984** [comment]
I wrote the qpid piece too, so I think that is good to go as well, looking at this again.

File: katello/hooks/post/09-mongo_syspath.rb [hunk 1]
```diff
@@ -1,17 +0,0 @@
-def install_syspaths
-  if facts[:os][:release][:major] == '7'
-    `rpm -qa | grep rh-mongodb34-syspaths`
-    if $?.success?
-      logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
-    else
-      logger.info 'rh-mongodb34-syspaths not present, installing.'
-      `yum install -y -q rh-mongodb34-syspaths`
```
Comment (ekohl) [inline katello/hooks/post/09-mongo_syspath.rb:8]: I think we still need to ensure this is installed. IIRC that can be done by setting the `mongodb::globals::server_package_name` to that in `config/foreman.hiera/family/RedHat-7.yaml`

**Review by ekohl — CHANGES_REQUESTED** [review]
I'm inclined to also trim down `upgrade_step` to remove tracking of all steps since it now always calls it with `:run_always => true`. We can also do that in a later commit. If we need it in the future, I actually want to use the mechanisms the installer has built in rather than this custom solution.

**Comment by ehelms** [comment]
Right now, I am planning to add a step for Postgres upgrade but I thought cleaning up first would be nice.

**Review by ekohl — APPROVED** [review]
Let's have a look at how the postgres one turns out. However, I'd like to use the native mechanism (`store_custom_config` and `get_custom_config`). See https://github.com/theforeman/foreman-installer/blob/5676453ec3226dff221cc1589a7118042fcdb843/katello/hooks/pre_commit/09-version_locking.rb on how to do that.

Of course that's all for the next PR and I'm now happy with this cleanup :)

**Comment by ekohl** [comment]
Of course just now I realize that this could have used a Redmine issue to track. Please do one for the postgres upgrade :)

**Pull request merged**